### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by ScribbleTAS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,67 @@
-﻿# LoTAS Mod
-### Low Optimization Tool Assisted Speedrun Mod
+﻿# This project is on hiatus!
```
Comment (PancakeTAS) [inline README.md]: can we please use words that everyone can understand...

File: README.md [hunk 2]
```diff
@@ -1,32 +1,67 @@
...
+We are focussing our attention to [TASmod](https://github.com/MinecraftTAS/TASmod).
```
Comment (PancakeTAS) [inline README.md]: "focu**s**ing" for international, also write it like this:
`We are shifting our focus to ...`

File: README.md [hunk 3]
```diff
@@ -1,32 +1,67 @@
-﻿# LoTAS Mod
-### Low Optimization Tool Assisted Speedrun Mod
+﻿# This project is on hiatus!
+We are focussing our attention to [TASmod](https://github.com/MinecraftTAS/TASmod).  
+For more information and plans read this announcement: [Future of Mineraft TAS tools](https://gist.github.com/ScribbleTAS/43b18e8a92a111e92bf4bb0d9cf49e99)
 
-LoTAS is a Mod used to create [Tool Assisted Speedruns](https://en.wikipedia.org/wiki/Tool-assisted_speedrun). In Minecraft TASing you use a mod called Tickrate Changer to slow the game down. The Game runs at 20 ticks per second so 10 would be 50% speed.
+# LoTAS
+Used to create non-playback, Low optimization Tool-Assisted Speedruns (LoTAS).  
+Essentially, you record the game in slow motion, record yourself speedrunning with a recording program such as [OBS](https://obsproject.com),
```
Comment (PancakeTAS) [inline README.md]: weird sentence, also it's not speedrunning cuz that sounds suspicously like RTA running.
`You essentially play through your route in slow motion while using a recording program such as [OBS] to capture yourself. Later you speed up the video footage and edit your attempts into a final video.`

File: README.md [hunk 4]
```diff
@@ -1,32 +1,67 @@
...
+then speed up the video footage.  
 
-## How do I start?
+LoTAS is a collection of tools commonly used for TASing, with a big focus on RNG manipulation.  
 
-Once you installed the Mod and opened a World you can access the TAS Features by pressing `Escape`.
-This might look confusing to you, but all you need to know is that the top left buttons + and -, increase or decrease the speed of the game.
-Have a look in the [Wiki](https://github.com/MCPfannkuchenYT/LoTAS/wiki) to see the rest of the features explained
+## Tools
+- *Tickratechanger*: Used for slowing down the game speed
```
Comment (PancakeTAS) [inline README.md]: it's Tickrate Changer, not Tickratechanger - at least in lotas

File: README.md [hunk 5]
```diff
@@ -1,32 +1,67 @@
...
+- *Savestates*: Used for creating backups of your world, basically quick save/quick load
+- *Drop Manipulation*: Manipulates mob drops to always drop the best items
+- *Dragon Manipulation*: Manipulates the dragon phases
+- *Item Duplication (Dupemod)*: Replicates the duping bug from vanilla, without the need to close the game.
+- *Spawn Manipulation*: Spawn entities at a certain location.
+- *AI Manipulation*: Let entities pathfind to a certain place  
 
+*and more...*
 ## Bugs / Feature Requests
+We won't accept any feature-requests at this time, as well as bug reports.
```
Comment (PancakeTAS) [inline README.md:25]: `Due to lack of time and motivation, we will, for the time being, not be accepting feature requests or bug reports`

File: README.md [hunk 6]
```diff
@@ -1,32 +1,67 @@
...
-* Savestates: Create copies of your World and load them again without copying the world manually!
-* Drop Manipulation: Need Flint? Get Flint!
-* Spawn Manipulation: Spawn Mobs everywhere possible using the Gui!
-* AI Manipulation: Change what the Dragon should do and where your Entities should move.
-* And More: There are more features like: Tick Indicator, Timer, No Damage and more.. all of those in the wiki.
+# FAQ
+## How do I start?
+- Most functionality is in the in-game pause menu via buttons.
+- The main menu has a "configuration" button
+- Pressing F6 opens an in game HUD, where you can enable information or an in-game timer.
```
Comment (PancakeTAS) [inline README.md]: - Most of LoTAS is accessible via the in-game pause menu.
[...]
- Pressing F6 [...], where you can display various informations onscreen.

File: README.md [hunk 7]
```diff
@@ -1,32 +1,67 @@
...
+1. Open the pause menu and click in the top left **Save Items**
+2. Throw item(s) on the ground
+3. Open the pause menu and click in the top left **Load Items**
+4. An item should spawn *in your inventory*
+
+### 1.8.9-1.12.2
+1. Throw the item(s) you want to duplicate on the ground
+2. Open the pause menu and click in the top left **Save Items**
+3. Pick up the item(s)
+4. Open the pause menu and click in the top left **LoadItems**
```
Comment (PancakeTAS) [inline README.md]: space missing

File: README.md [hunk 8]
```diff
@@ -1,32 +1,67 @@
...
+5. An item should spawn *on the ground*
+
+## Where is the video upspeeder?
+This feature that was a minor desaster...
```
Comment (PancakeTAS) [inline README.md:64]: i hate you <3

**Review by PancakeTAS — CHANGES_REQUESTED** [review]
I don't like that it just links TASmod that often. We need to add one of those new markdown tags to the TASmod README.md at the very top. Here:
```
> [!WARNING]
> This mod is in **early-development** and is not ready for use yet. If you wish to download the early testing releases, be prepared to encounter various bugs and desyncs!
```
Now people won't download TASmod thinking it's a fully working mod.

File: README.md (hunk 1 above)
Comment (ScribbleTAS) [reply README.md]: Hiatus is relatively common... I was also thinking about "hibernation" but thats also ehh... Not maintained sounds too harsh? idk

Comment (PancakeTAS) [reply README.md]: `This project is no longer actively being developed!`

File: README.md (hunk 5 above)
Comment (ScribbleTAS) [reply README.md:25]: This does sound like: We are bored of you, don't message us...

Comment (PancakeTAS) [reply README.md:25]: i think it's fine tbh

File: README.md (hunk 8 above)
Comment (ScribbleTAS) [reply README.md:64]: ♥

**Comment by ScribbleTAS** [comment]
> We need to add one of those new markdown tags to the TASmod README.md at the very top.

Feel free to make a PR ♥

**Comment by PancakeTAS** [comment]
hold on let me rephrase that - I'm not approving this pr until that notice is added.

**Review by PancakeTAS — APPROVED** [review]
